_specs/ecip-1072.md

```diff
@@ -1,10 +1,10 @@
 ---
 lang: en
 ecip: 1072
-title: Yingchun EVM and Protocol Upgrades
+title: Aztlán EVM and Protocol Upgrades (Yingchun Edition)
 status: Draft
 type: Meta
-author: Wei Tang (@sorpaas)
+author: Wei Tang (@sorpaas), Talha Cross (@soc1c), Bob Summerwill (@bobsummerwill)
```

Comment:
ref ECIP-1061 and #186

Reply:
That is very kind, @sorpaas, but there is really no need to add me as an author.